homeassistant/components/switch/isy994.py

```diff
@@ -0,0 +1,82 @@
+""" Support for ISY994 lights. """
+# system imports
+import logging
+
+# homeassistant imports
+from homeassistant.components.isy994 import (ISY, ISYDeviceABC, SENSOR_STRING,
+                                             HIDDEN_STRING)
+from homeassistant.const import STATE_ON, STATE_OFF  # STATE_OPEN, STATE_CLOSED
+# The frontend doesn't seem to fully support the open and closed states yet.
+# Once it does, the HA.doors programs should report open and closed instead of
+# off and on. It appears that on should be open and off should be closed.
```

Comment:
Just out if curiousity, do you mean with a door an actual door that you can open and close from ISY?